net/netsniff-ng/Makefile

```diff
@@ -0,0 +1,68 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=netsniff-ng
+PKG_VERSION:=0.6.7
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/netsniff-ng/netsniff-ng/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=030b0554206b43c5883e48a63575fb65d7835e275333372f55f7d615d419da05
+
+PKG_USE_MIPS16:=0
+
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/netsniff-ng
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+libpthread +libpcap +libncurses +zlib +liburcu +libsodium +libnetfilter-conntrack
+  TITLE:=netsniff-ng
+  MAINTAINER:=Toni Uhlig <matzeton@googlemail.com>
```

Comment:
I'd move this above PKG_LICENSE and rename to PKG_MAINTAINER.

Reply:
done